spec/automation/unit/method_validation/dialog_parser_spec.rb

```diff
@@ -91,7 +91,13 @@ def load_tags
     it "with no dialogs set" do
       @root_stp.options = @root_stp.options.merge(:dialog => {})
       @root_stp.save
-      expect { run_automate_method }.to raise_exception
+      expect { run_automate_method }.not_to raise_exception
```

Comment:
@tinaafitz 
Is this a change in behavior, do we have to worry about backward compatibility?